src/index.html

```diff
@@ -4,10 +4,302 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Miami</title>
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
+    <script type="text/javascript" src="./scripts/main.js"></script>
   </head>
   <body>
-    <h1>Miami</h1>
-    <script type="text/javascript" src="scripts/main.js"></script>
+    <header class="header" id="home">
+      <div class="header__content">
+        <div class="header__head">
+          <input
+            class="header__nav-toggler"
+            id="nav-toggler"
+            type="checkbox"
+          >
+          <label class="header__nav-button" for="nav-toggler"></label>
+
+          <a href="#home"><div class="header__logo"></div></a>
+          <a class="header__phone-link" href="tel:+12345555555">
+            <div class="header__phone"></div>
+          </a>
+
+          <nav class="header__nav-burger">
+            <div class="header__nav-content">
+              <ul class="header__nav-list">
+                <li class="header__nav-item">
+                  <a href="#home" class="header__nav-link">home</a></label>
+                </li>
+                <li class="header__nav-item">
+                  <a href="#about-us" class="header__nav-link">about us</a>
+                </li>
+                <li class="header__nav-item">
+                  <a href="#gallery" class="header__nav-link">gallery</a>
+                </li>
+                <li class="header__nav-item">
+                  <a href="#" class="header__nav-link">consultation</a>
+                </li>
+                <li class="header__nav-item">
+                  <a href="#testimonials" class="header__nav-link">testimonials</a>
+                </li>
+                <li class="header__nav-item">
+                  <a href="#contacts" class="header__nav-link">contacts</a>
+                </li>
+              </ul>
+
+              <a class="header__nav-phone-link" href="tel:+12345555555">
+                <p class="header__nav-phone">+1 234 555-55-55</p>
+                <p class="header__nav-phone">call to order</p>
+                <div class="header__nav-line"></div>
+              </a>
+            </div>
+          </nav>
+        </div>
+
+        <div class="header__title">
+          <div class="header__line"></div>
+          <h1 class="header__text">Miami<br>Condo Kings</h1>
+        </div>
+
+      </div>
+    </header>
+
+    <main class="main-content">
```

Comment:
```suggestion
    <main class="main main__content">
```
You can use a mix of classes. It's useful because you are giving margins to your `.main-content`, but it's not allowed, according to BEM. (`main-content` here is a BEM-block)